src/gnatss/utilities/time.py

```diff
@@ -4,23 +4,20 @@
 """
 from astropy.time import Time as AstroTime  # noqa
 from astropy.time.formats import TimeFromEpoch, erfa
+from astropy.units import isclose
 
+__all__ = ["AstroTime", "erfa", "isclose"]
 
-class TimeUnixJ2000(TimeFromEpoch):
 
+class TimeUnixJ2000(TimeFromEpoch):
     """
-
-    Seconds from 2000-01-01 12:00:00 UTC
-
+    Seconds from 2000-01-01 12:00:00 TT (equivalent to 2000-01-01 11:58:55.816 UTC),
+    ignoring leap seconds.
     """
 
     name = "unix_j2000"
-
     unit = 1.0 / erfa.DAYSEC  # in days (1 day == 86400 seconds)
-
-    epoch_val = "2000-01-01 12:00:00"
-
+    epoch_val = "2000-01-01 11:58:55.816"
```

Comment:
Thanks for making these changes @madhavmk. A little background with this python package is that this is a port from a Fortran program, so I need to ensure the outputs are currently matching. Therefore, it looks like one issue with using the `utc` scale here is that when I call `mytime.iso` I get the incorrect value compared to the Fortran one. For example:

```python
aj2000 = AstroTime(712215465.0, format="unix_j2000")

print(aj2000.iso) # 2022-07-27 17:36:40.816
```

That's what your change would output. However, If I keep the `epoch_val` the way it is and change the scale to be `tt` and do the same thing, I get the correct value of `2022-07-27 17:37:45.000`. If I need utc iso, I can simply do `aj2000.utc.iso`. This is why i suggested this way in the comment: https://github.com/uw-ssec/offshore-geodesy/issues/108#issuecomment-1661005108

Reply:
You're right. Using `tt` scale matches the fortran library output. I have added a commit updating this PR adding the testcases you provided below.
p.s. Is the fortran module open-sourced? I am interested in checking it out!

Reply:
>  p.s. Is the fortran module open-sourced? I am interested in checking it out!

The fortran is not open-source unfortunately.